api/internal/messenger/entity/contact_read.go

```diff
@@ -0,0 +1,40 @@
+package entity
+
+import (
+	"time"
+
+	"github.com/and-period/furumaru/api/pkg/uuid"
+)
+
+type ContactRead struct {
+	ID        string    `gorm:"primaryKey;<-:create"` // お問い合わせ会話履歴ID
+	ContactID string    `gorm:""`                     // お問い合わせID
+	UserID    string    `gorm:"default:null"`         // 送信者ID(ゲストの場合null)
+	UserType  int32     `gorm:""`                     // 送信者の種別(不明:0, admin:1, user:2, guest:3)
+	ReadFlag  bool      `gorm:""`                     // 既読フラグ
```

Comment:
DDL見るとFlagは必要なさそうかも..?👀

```suggestion
	Read  bool      `gorm:""`                     // 既読フラグ
```